packages/dataviews/README.md

```diff
@@ -318,7 +318,15 @@ The `defaultLayouts` property should be an object that includes properties named
 
 ### `onChangeSelection`: `function`
 
-Callback that signals the user selected one of more items, and takes them as parameter. So far, only the `list` view implements it.
+Callback that signals the user selected one of more items, and takes them as parameter.
+
+### `isItemClickable`: `function`
```

Comment:
Is this one really necessary? I mean what happens if we just do nothing in `onClickItem` instead?

Reply:
> Is this one really necessary? I mean what happens if we just do nothing in onClickItem instead?

This can be useful for differentiating the layout style of clickable primary fields from non-clickable ones. In the page view, the “primary” field is gray when the item is not clickable:


https://github.com/user-attachments/assets/2fdc2b6f-5a94-4710-ac37-4738dbc970ae


(also, I just noticed that `deleted` pages aren't visible in the `All pages` view)